.github/workflows/_local_ci_cd_updated_main.yml

```diff
@@ -16,8 +16,9 @@ jobs:
       default_repo_branch: main
       update_docs: true
       doc_extras: "[docs]"
-      update_python_api_ref: false
+      update_python_api_ref: true
```

Comment:
Does this have anything to do with the issue?

Reply:
Ok, it doesn't really. This is just adding documentation here. I don't understand why that is the same issue about allowing for more detailed control of major and minor version updates for dependencies, though.

Reply:
It isn't. It's completely separate, really :) I just did this in the previous PR, hence I separated it out to create this separate PR.